co2eq/food/ingredients.js

```diff
@@ -32,11 +21,10 @@ export const explanation = {
 
 export const modelCanRunVersion = 1;
```

Comment:
should be bumped

Reply:
what do you mean? There was no ingredients model before?

Reply:
ah got it - my bad!